chef.gemspec

```diff
@@ -25,7 +25,10 @@ Gem::Specification.new do |s|
   s.add_dependency "ohai", ">= 8.6.0.alpha.1", "< 9"
 
   s.add_dependency "ffi-yajl", "~> 2.2"
-  s.add_dependency "net-ssh", ">= 2.9", "< 4.0"
+  s.add_dependency "net-ssh", "~> 4.0"
```

Comment:
this should probably be `s.add_dependency "net-ssh", ">= 2.9", "< 5.0"` to avoid fencing problems with gems that are still `"~> 3.0"`